index.html

```diff
@@ -92,16 +92,16 @@ <h1>
     </section>
     <section id="recent-works-start" class="recent-works">
       <div class="section-heading flex">
-        <h1>My Recent Works</h1>
+        <h2>My Recent Works</h2>
         <div class="indicator"></div>
       </div>
       <div class="card-container">
         <div class="card flex">
           <img
             src="https://images.unsplash.com/photo-1618477247222-acbdb0e159b3/?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=700&q=80"
-            class="image" alt="placeholder" />
+            class="image" alt="" />

```

Comment:
- [ ] Great work adding the alt property as default. Now, let us add the appropriate alternative text, that's useful for interacting or understanding the page